src/test/resources/docker/edb-init.sql

```diff
@@ -0,0 +1,53 @@
+-- Script to create enterprisedb user and database 
+
+SET AUTOCOMMIT ON; 
+  
+CREATE USER lbuser WITH PASSWORD 'LiquibasePass1';
+CREATE DATABASE "lbcat";
+GRANT ALL PRIVILEGES ON DATABASE lbcat TO lbuser;
+--GRANT ALL PRIVILEGES ON SCHEMA public TO lbuser;
+\connect lbcat;
```

Comment:
is this slash needed here or it's a typo? Looks weird for me

Reply:
Yes, the slash is needed. Within psql, `\connect` or the shortened `\c` is how you switch to another database.